### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Vital-Vuillaume** [Vital-Vuillaume/My-Website#3] [Announcements]
You advised me to start with the mobile or computer size?

**Comment by judemont** [comment]
Computer I think... But i dont know..

**Comment by judemont** [comment]
I have a question,
I have seen that you have restarted your website. Can you explain me what is your project in this website ?

**Comment by Vital-Vuillaume** [comment]
I will make it a page of the site where I explain what I do. In the second page you can look for my projects and in the third page there is a space where there will be several parameters it is mainly to train me to do JavaScript.

**Comment by Vital-Vuillaume** [comment]
test
